README.md

```diff
@@ -42,6 +42,17 @@ If you have Go 1.17+, you can setup a development environment:
     $ go install ./cmd/cosign
     $ $(go env GOPATH)/bin/cosign
 
+## Dockerfile
+Here is how to install and use cosign inside a Dockerfile through the gcr.io/projectsigstore/cosign image:
+
+```shell
+FROM gcr.io/projectsigstore/cosign:v1.9.0 as cosign-bin
+
+FROM scratch
```

Comment:
This will work, but then will the image actually be usable? You'd probably need ca-certs as well to actually talk to any registries at a minimum. Maybe we should use distroless in the example?

Reply:
That makes a lot of sense, let me update it before merge, thanks 🙏

Reply:
I think we are good with: acd342ef8bd0ea206137b781fb1fc251e7316cd7